src/utils/index.ts

```diff
@@ -217,8 +217,9 @@ export const calculatePercentage = (value: number, percentage: number): number =
 }
 
 export const switchOrAddNetwork = (networkDetails?: NetworkDetails, account?: string) => {
-  if (!window.ethereum || !window.ethereum.request || !window.ethereum.isMetaMask || !networkDetails || !account) return
-  window.ethereum
+  if (!window.ethereum || !window.ethereum.request || !window.ethereum.isMetaMask || !networkDetails || !account)
```

Comment:
this (almost) the same condition as above, with account verification.

https://github.com/levelkdev/swapr-dapp/pull/1438/files#diff-6c91347fbfbee4686763a65f4ca7bf947d87ef6ab5dd86772fd6cd4c734a9043R133

Maybe there is an opportunity to share the same variable?